steps/microsoft-teams-integration/1.2.2/step.yml

```diff
@@ -0,0 +1,139 @@
+title: Microsoft Teams Integration
+summary: Allows you to recieve build status updates on Microsoft Team using Incoming
+  Webhooks
+description: |
+  Uses Incoming Webhooks feature in Microsft Teams to send updates on your builds
+website: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+source_code_url: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+support_url: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration/issues
+published_at: 2021-02-18T21:28:22.072847+11:00
+source:
+  git: https://github.com/amrfarid140/bitrise-step-microsoft-teams-integration.git
+  commit: 83a518dc014f7bfcbfff1104aaafd7fca0600824
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/amrfarid140/bitrise-step-microsoft-teams-integration
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: true
+inputs:
+- card_title: $BITRISE_APP_TITLE
+  opts:
+    description: |
+      Primary MessageCard title text.
+
+      If no value is given, the default value will be the `BITRISE_APP_TITLE`.
+    summary: The heading text at the top of the MessageCard
+    title: Card Title
+- enable_debug: false
+  opts:
+    description: |
+      Enables debug logging to the Bitrise console.
+
+      WARNING: This will output the JSON value of the Teams Webhook body, be mindful of what is output here.
+    summary: Toggles whether to log debug info to the build output
+    title: Enable debug logging
+- opts:
+    category: Theme Customisation
+    description: |
+      Given value should be a hexadecimal color, minus the leading `#`.
+
+      Defaults to `359914` if not overridden.
+    summary: Sets the cards theme color to the given hex value for successful builds
+    title: Theme Color - Build Success
+  success_theme_color: 359914
+- failed_theme_color: E04504
+  opts:
+    category: Theme Customisation
```

Comment:
All other steps use US/Canadian spelling: "Customization" (s -> z).

Reply:
This isn't new as it's been in here for several versions now... no one had a problem with it when I initially added this and it was approved?

Reply:
thanks for the contribution but no official requirement is present at the moment for steps to use specific locales